testing/test_collection.py

```diff
@@ -172,17 +172,6 @@ def pytest_collect_directory(self, path, parent):
         assert "world" in wascalled
 
 class TestPrunetraceback:
-    def test_collection_error(self, testdir):
-        p = testdir.makepyfile("""
-            import not_exists
-        """)
-        result = testdir.runpytest(p)
-        assert "__import__" not in result.stdout.str(), "too long traceback"
```

Comment:
This test was here to explicitly cut back the traceback from the error message, but unfortunately this makes it more difficult to find bugs.